histogram/src/standard.rs

```diff
@@ -278,18 +278,27 @@ impl<'a> Iterator for Iter<'a> {
     }
 }
 
+impl From<&SparseHistogram> for Histogram {
+    fn from(other: &SparseHistogram) -> Self {
+        let mut histogram = Histogram::with_config(&other.config);
+
+        for bucket in other.into_iter() {
+            // This can't fail because both histograms have the same config
+            let _ = histogram.add(bucket.start(), bucket.count());
```

Comment:
Not a huge concern, but wouldn't this be easier to just use `as_mut_slice` and then just set the value of the buckets directly?

Reply:
Or even just a regular `Vec<u64>` and then construct using `from_buckets`, I guess

Reply:
Good catch. Changed to use iterators across other and set the values directly.